pkg/licensing/expression/types.go

```diff
@@ -0,0 +1,68 @@
+package expression
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/aquasecurity/trivy/pkg/licensing"
+)
+
+var versioned = []string{
+	licensing.AGPL10,
+	licensing.AGPL30,
+	licensing.GPL10,
+	licensing.GPL20,
+	licensing.GPL30,
+	licensing.LGPL20,
+	licensing.LGPL21,
+	licensing.LGPL30,
+}
```

Comment:
SPDX docs contain `GFDL` licenses with `-or-later/-only` suffix(e.g. GFDL-1.1-invariants-only/GFDL-1.1-invariants-or-later).
`licenseclassifier` doesn't contain these licenses in types(https://github.com/google/licenseclassifier/blob/7c62d6fe8d3aa2f39c4affb58c9781d9dc951a2d/license_type.go#L24-L177).
But we can take licenses not only from `licenseclassifier`(e.g. dpkg copyright files).

Looks like we also need to add their.

Reply:
I knew it, but I was lazy 😄 
https://github.com/aquasecurity/trivy/pull/3824/commits/c38b5fafb7fe1792ca3d50c0dfe70e08e45cb68f